packages/salesforcedx-vscode-core/src/commands/templates/forceFunctionCreate.ts

```diff
@@ -137,14 +126,39 @@ export class FunctionInfoGatherer implements ParametersGatherer<FunctionInfo> {
     return {
       type: 'CONTINUE',
       data: {
-        fileName: name,
         language
       }
     };
   }
 }
 
+function nameVerification(value: string) {
+  if (value.length > 47) {
+    throw new Error('Function names cannot contain more than 47 characters.');
+  }
+  const functionNameRegex = /^[a-z0-9]+$/;
+  if (!functionNameRegex.test(value)) {
+    throw new Error('Name must contain only lowercase letters and numbers.');
+  }
+  const functionNameStartRegex = /^[a-z]/;
+  if (!functionNameStartRegex.test(value)) {
+    throw new Error('Name must start with a letter.');
+  }
+  return '';
+} // TODO move to appropriate class/package and/or localize strings
```

Comment:
Not sure where this function belongs. As a part of this file/class? `@salesforce/templates`? `@heroku/functions-core`?

Reply:
Also need to localize strings